internal/controller/operator/factory/build/common.go

```diff
@@ -4,13 +4,43 @@ import (
 	"context"
 	"fmt"
 	"path"
+	"sort"
 
 	vmv1beta1 "github.com/VictoriaMetrics/operator/api/operator/v1beta1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+type keysSorter[T any] struct {
+	target []T
+	sorter []string
+}
+
+func OrderByKeys[T any](target []T, sorter []string) {
```

Comment:
This function requires proper comment.

 Also, it's good practice to write comments for any public exported methods.

Reply:
added comment